index.js

```diff
@@ -220,7 +220,15 @@ function isRequestOriginAllowed (reqOrigin, allowedOrigin) {
   }
 }
 
-module.exports = fp(fastifyCors, {
+const _fastifyCors = fp(fastifyCors, {
```

Comment:
revert tiese changes, and I will approve and merge it ;)

Reply:
This addresses the issue I found in https://github.com/akphi/issue-repo/pull/11 in my project using `NodeNext`. Is there something I miss here, but what you said about `fastify-plugin` just doesn't work for our Typescript project.